org_status/status_providers/travis.py

```diff
@@ -16,3 +17,11 @@ def get_status(self, repo, host, branch='master'):
             return Status.UNDETERMINED
 
         return self.status_from_badge_svg(badge_result.text)
+
+    @classmethod
+    def get_system_status():
```

Comment:
Since this is a class method, it should take in one parameter: `cls`, which is a reference to the class.